prompt_toolkit/shortcuts/progress_bar/formatters.py

```diff
@@ -76,7 +77,9 @@ class Label(Formatter):
     :param suffix: String suffix to be added after the task name, e.g. ': '.
         If no task name was given, no suffix will be added.
     """
-    def __init__(self, width: AnyDimension = None, suffix: str = '') -> None:
+    def __init__(self, width: AnyDimension = None, suffix: str = '', *,
+                 name: Optional[str] = None) -> None:
+        self.name = operator.attrgetter(name or self.__class__.__name__.lower())
```

Comment:
This line can then be:
```python
self.name = name
```
(I prefer that attributes, received in ``__init__`` are assigned to self, unmodified if possible. I know that we are violating this sometimes with the ``to_filter`` stuff, but I think there's no need here.)